libmachine/cert/cert.go

```diff
@@ -195,12 +207,15 @@ func ValidateCertificate(addr, caCertPath, serverCertPath, serverKeyPath string)
 		return false, ErrValidatingCert{err}
 	}
 
+	log.Debugf("Configuration: +%v", tlsConfig)
+
 	dialer := &net.Dialer{
 		Timeout: time.Second * 2,
 	}
 
 	_, err = tls.DialWithDialer(dialer, "tcp", addr, tlsConfig)
 	if err != nil {
+		log.Debugf("Certs are not valid: %s", err)
```

Comment:
@dgageot @nathanleclaire We should probably keep this debugging line in the release? It helps debugging when people are not capable of building some debugging PR by posting those lines...